server.js

```diff
@@ -45,7 +46,7 @@ const tracks = {
 
 app.post('/push-release/:tag/:commit', handleAsync(async function (req, res) {
 	if (keccak256(req.body.secret || '') !== secretHash) {
-		throw new Error('Bad request');
+		throw new Error('Invalid secret');
```

Comment:
Can this bit be a middleware if it's used several times?

Reply:
Had the same idea, realized in https://github.com/paritytech/push-release/pull/8